lib/storage/src/types.rs

```diff
@@ -78,6 +81,10 @@ fn default_snapshots_path() -> String {
     "./snapshots".to_string()
 }
 
+fn default_tmp_path() -> String {
+    "./storage".to_string()
```

Comment:
I don't know if this was addressed, but there's a subtle difference from the original behavior.

Originally, if we set a storage path on the settings, the tmp_dir will default to the new storage path. (e.g., storage and tmp both point to `/foo`)

After the change, if we set a storage path _but not_ tmp_path: tmp_path will default to `/storage` while the storage path will be elsewhere (e.g. `/foo`)

This might surprise unaware users if they plan not to use `/storage` altogether (and might lead to permission issues since we're using `create_dir_all`)

Reply:
You're right!

I suggested a slightly different approach, which would also resolve this.

https://github.com/qdrant/qdrant/pull/1960#pullrequestreview-1450487730

Reply:
Hi, 

Thanks for the reviews. I've taken the suggested changes into the PR
Let me know what you think.